### DISCUSSION TITLE
photoprism: ERROR 2@134 Unknown failure

### DISCUSSION THREAD

**Discussion by rechtsanwalt-petter** [tteck/Proxmox#534] [Help Needed] [answered]
During the photoprism skript it gets stuck with the following error:

` - Building PhotoPrism (Patience)...‼ ERROR 2@134 Unknown failure occurred.`

I tried different settings, but do not get past this point. Does anyone have a hint, why this happens?

Thanks in advance.
Cheers!

**Comment by tteck** [comment]
Have you tried adding more cores?

**Comment by rechtsanwalt-petter** [comment, marked as answer]
Thank you for the quick hint. Simply raising the core number did not work. But more cores combined with an unpriviliged container solved the problem in the end.

Really amazing work with your skripts. Therefore, thanks a lot! :)
